.github/dependabot.yml

```diff
@@ -14,5 +14,3 @@ updates:
     open-pull-requests-limit: 5
     commit-message:
```

Comment:
IMO the whole `commit-message` section can be removed.

Reply:
I don't have strong feelings either way.  I just figured some way of separating dev from packaged dependencies would be nice.  Wish dependabot would just label it as such.

Reply:
Dependabot does add labels

Reply:
Not in a way that allows you to filter development tendencies from production ones. It only adds the "dependencies" label, and another label based on the package manager.

Reply:
I don't think that matter at all, all requirement changes needs to be validated the same way regardless if the are nested under "devRequirements" or not.

Reply:
Fair enough. I am away from my computer for a bit, so feel free to push the change to remove it all.